src/Package/Impl/Packages/R/RPackage.cs

```diff
@@ -112,23 +112,27 @@ internal class RPackage : BasePackage<RLanguageService>, IRPackage {
             VsWpfOverrides.Apply();
             CranMirrorList.Download();
 
+            base.Initialize();
+
+            ProjectIconProvider.LoadProjectImages();
+            LogCleanup.DeleteLogsAsync(DiagnosticLogs.DaysToRetain);
+
+            IdleTimeAction.Create(CompleteInit, 20, this.GetType(), VsAppShell.Current);
```

Comment:
The package will be loaded since initialization will be complete. But it depends what exactly project uses from the package. Appears to work in real life, but that's just one machine. I assume we are talking project reopen with workspace auto-load?